environments/dev/backend_lb_config.yaml

```diff
@@ -6,7 +6,7 @@ gateways:
       ssl_host_name_suffix: dev.platform.hmcts.net
       certificate_name: wildcard-dev-platform-hmcts-net
     app_configuration:
-    # Toffee
+    # Toffee application
```

Comment:
isn't there a parameter in the pipeline to ignore these flags?

Reply:
files_list: 'components/${{ parameters.stack }} environments/${{ parameters.environment }} environments/variable.tf'
Is file is located under the  "environments/${{ parameters.environment }}" along with tfvars

I just need to want the apply job in the pipeline to run